### PR TITLE
Updates to support Verizon Media SDK v1.4.0

### PR DESCRIPTION
**Pull request by jonalter**

### CODE REVIEW

File: ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ apply plugin: 'maven-publish'
  */
 ext {
     // String property to store version name.
-    stringVersion = "1.3.1.0"
+    stringVersion = "1.4.0.0"
     // String property to store group id.
     stringGroupId = "com.google.ads.mediation"
 }
```
Comment (ericleich) [inline ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle]: Can you also update versionCode to be 1040000?

Comment (jonalter) [reply ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle]: Gladly. I missed that.

Comment (jonalter) [reply ThirdPartyAdapters/verizonmedia/verizonmedia/build.gradle]: ✔️

**Comment by brownEugene** [comment]
Verified VerizonMedia Adapter supports VerizonMedia SDK 1.4.0

**Pull request merged**